src/c++/perf_analyzer/test_concurrency_manager.cc

```diff
@@ -156,7 +236,21 @@ class TestConcurrencyManager : public TestLoadManagerBase,
     CheckSequences(concurrency2);
   }
 
+  std::shared_ptr<ModelParser>& parser_{LoadManager::parser_};
+  std::shared_ptr<DataLoader>& data_loader_{LoadManager::data_loader_};
+  bool& using_json_data_{LoadManager::using_json_data_};
+  bool& execute_{ConcurrencyManager::execute_};
+  size_t& batch_size_{LoadManager::batch_size_};
+  size_t& max_threads_{LoadManager::max_threads_};
+  bool& async_{LoadManager::async_};
+  bool& streaming_{LoadManager::streaming_};
```

Comment:
Are this and `async_` needed?

Reply:
nope. i removed them. thanks for catching that.